deploy/terraform/variables.tf

```diff
@@ -0,0 +1,77 @@
+
+
+variable "etcd_image" {
+  type        = string
+  default     = "docker.io/bitnami/etcd:3.4.15-debian-10-r43"
+  description = "etcd image to use"
+}
+
+variable "nats_image" {
+  type        = string
+  description = "amount of hugepages to allocate for mayastor"
+  default     = "nats:2.2.6-alpine3.13"
+}
+
+variable "msp_operator_image" {
+  type        = string
+  description = "msp operator image to use"
+  default     = "mayadata/mcp-msp-operator:develop"
+}
+
+variable "rest_image" {
+  type    = string
+  default = "mayadata/mcp-rest:develop"
+}
+
+variable "core_image" {
+  type    = string
+  default = "mayadata/mcp-core:develop"
+}
+
+variable "mayastor_image" {
+  type        = string
+  description = "mayastor image to use"
+  default     = "192.168.1.4:5000/mayadata/mayastor:latest"
```

Comment:
Looks like you're using a local repo here.

Reply:
Can we customize it to take a local registry and alternative image tag as options?

Reply:
Yeah we can do var.registry and var.tag and use format()